internal/pkg/v1/action/action_suite_test.go

```diff
@@ -63,3 +65,69 @@ func newClusterCatalog(name string) *olmv1.ClusterCatalog {
 		ObjectMeta: metav1.ObjectMeta{Name: name},
 	}
 }
+
+type extensionOpt func(*olmv1.ClusterExtension)
+
+func withVersion(version string) extensionOpt {
+	return func(ext *olmv1.ClusterExtension) {
+		ext.Spec.Source.Catalog.Version = version
+	}
+}
+
+func withSourceType(sourceType string) extensionOpt {
+	return func(ext *olmv1.ClusterExtension) {
+		ext.Spec.Source.SourceType = sourceType
+	}
+}
+
+// nolint: unparam
```

Comment:
Which lint error is this masking?

Reply:
internal/pkg/v1/action/action_suite_test.go:83:27  unparam  `withConstraintPolicy` - `policy` always receives `string(olmv1.UpgradeConstraintPolicyCatalogProvided)` (`"CatalogProvided"`)

I don't think this needs to be 'fixed', hence `nolint`. `withConstraintPolicy` is used only to build test operators (extension) in tests. We do have test cases where we check setting a different value, but the base remains the same.